HotFix.Test/utilities/writing/datetimes.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Text;
+using FluentAssertions;
+using HotFix.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace HotFix.Test.utilities.writing
+{
+    [TestClass]
+    public class datetimes
+    {
+        private byte[] _buffer;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            _buffer = new byte[23];
```

Comment:
Should we add unit tests for overflowing the available buffer size, ie show what happens when the buffer size is less than the expected data